benchmarks/test_colander.py

```diff
@@ -0,0 +1,55 @@
+import pkg_resources
+import colander as c
+
+
+class TestColander:
+    package = "colander"
```

Comment:
single quotes please. This code isn't linted yet, but pydantic prefers single quotes everywhere else.

Reply:
and in other places.